<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClutterUI</class>
 <widget class="QWidget" name="ClutterUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clutter Base</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="db_view">
     <property name="title">
      <string>DatabaseView</string>
     </property>
     <layout class="QVBoxLayout" name="db_layout"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="controls_gb">
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="select_db">
        <property name="text">
         <string>Select DB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
